@here/harp-mapview/lib/text/FontCatalogLoader.ts

```diff
@@ -39,7 +39,10 @@ export class FontCatalogLoader {
         return defaultFontCatalogName;
     }
 
-    async loadCatalogs(catalogCallback: FontCatalogCallback): Promise<void[]> {
+    async loadCatalogs(
+        catalogCallback: FontCatalogCallback,
+        failureCallback?: (name: string, error: Error) => void
+    ): Promise<void[]> {
```

Comment:
I understand that we log an error if we don't load the default font catalog.
Shouldn't we allow to build a MapView without default font catalog and don't log any error in that case?

Reply:
The FontCatalogLoader will only load a font, if there is text added to MapView, that needs a font, otherwise not error message is thrown